Tests/Bricks/ImageBrickTests.swift

```diff
@@ -51,6 +51,11 @@ class ImageBrickTests: XCTestCase {
         imageURL = NSURL(fileURLWithPath: NSBundle(forClass: self.classForCoder).pathForResource("image0", ofType: "png")!)
     }
 
+    override func tearDown() {
+        super.tearDown()
+        BrickCollectionView.imageDownloader = NSURLSessionImageDownloader() // Reset the image downloader back to the default one (if a test had overwritten this)
```

Comment:
OK, if that's the intent I'm alright with that. It triggers my global-variable aversion but this rationale makes sense.